docs/modules/ROOT/pages/api/erc1155.adoc

```diff
@@ -0,0 +1,664 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:eip1155: https://eips.ethereum.org/EIPS/eip-1155[EIP1155]
+:receiving-tokens: xref:/erc1155.adoc#receiving_tokens[Receiving Tokens]
+:inner-src5: xref:api/introspection.adoc#ISRC5[SRC5 ID]
+
+= ERC1155
+
+Reference of interfaces, presets, and utilities related to ERC1155 contracts.
+
+TIP: For an overview of ERC1155, read our xref:erc1155.adoc[ERC1155 guide].
+
+== Core
+
+[.contract]
+[[IERC1155]]
+=== `++IERC1155++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/cairo-2/src/token/erc1155/interface.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```javascript
+use openzeppelin::token::erc1155::interface::IERC1155;
+```
+Interface of the IERC1155 standard as defined in {eip1155}.
+
+[.contract-index]
+.{inner-src5}
+--
+0x6114a8f75559e1b39fcba08ce02961a1aa082d9256a158dd3e64964e4b1b52
+--
+
+[.contract-index]
+.Functions
+--
+* xref:#IERC1155-balance_of[`++balance_of(account, token_id)++`]
+* xref:#IERC1155-balance_of_batch[`++balance_of_batch(accounts, token_ids)++`]
+* xref:#IERC1155-safe_transfer_from[`++safe_transfer_from(from, to, token_id, value, data)++`]
+* xref:#IERC1155-safe_batch_transfer_from[`++safe_batch_transfer_from(from, to, token_ids, values, data)++`]
+* xref:#IERC1155-is_approved_for_all[`++is_approved_for_all(owner, operator)++`]
+* xref:#IERC1155-set_approval_for_all[`++set_approval_for_all(operator, approved)++`]
+--
+
+[.contract-index]
+.Events
+--
+* xref:#IERC1155-TransferSingle[`++TransferSingle(operator, from, to, id, value)++`]
+* xref:#IERC1155-TransferBatch[`++TransferBatch(operator, from, to, ids, values)++`]
+* xref:#IERC1155-ApprovalForAll[`++ApprovalForAll(owner, operator, approved)++`]
+* xref:#IERC1155-URI[`++URI(value, id)++`]
+--
+
+==== Functions
+
+[.contract-item]
+[[IERC1155-balance_of]]
+==== `[.contract-item-name]#++balance_of++#++(account: ContractAddress, token_id: u256) → u256++` [.item-kind]#external#
+
+Returns the amount of `token_id` owned by `account`.
+
+[.contract-item]
+[[IERC1155-balance_of_batch]]
+==== `[.contract-item-name]#++balance_of_batch++#++(accounts: Span<ContractAddress>, token_ids: Span<u256>) → Span<u256>++` [.item-kind]#external#
+
+Returns a list of balances derived from the `accounts` and `token_ids` pairs.
+
+[.contract-item]
+[[IERC1155-safe_transfer_from]]
+==== `[.contract-item-name]#++safe_transfer_from++#++(from: ContractAddress, to: ContractAddress, token_id: u256,  value: u256, data: Span<felt252>)++` [.item-kind]#external#
+
+Transfers `value` of `token_id` from `from` to `to`, checking first that `to` is aware of the ERC1155 protocol to prevent tokens being locked forever.
```

Comment:
```suggestion
Transfers `value` of `token_id` from `from` to `to`, checking first that `to` is able to handle ERC1155 tokens to prevent getting them locked forever.
```
i think the previous description excluded accounts